src/theme/components/featuredContent.ts

```diff
@@ -46,37 +47,53 @@ const FeaturedContent = defineMultiStyleConfig({
             bgColor: "dark.ui.bg.default",
           },
           img: screenreaderOnly(),
+          containerType: "inline-size",
+          "@container (min-width: 0px)": {
+            "[data-wrapper]": {
+              flexDirection: imageAtEnd ? "column-reverse" : "column",
+            },
+            "[data-image]": {
```

Comment:
This aligns with the `imageWrapper` styles below, right? Can we make it a rule that the same name be used across the container query styles and the standard styles?
```
data-imageWrapper
```

Reply:
Just rechecked this, data-wrapper refers to the wrapper of the image wrapper and text. data-image aligns to the imgWrapper styles